source/geod24/LocalRest.d

```diff
@@ -702,7 +719,7 @@ public final class RemoteAPI (API) : API
                         });
 
                     // now handle any leftover messages after any sleep() call
-                    if (!isSleeping())
+                    if (!terminated && !isSleeping())
```

Comment:
Why this clause ?

Reply:
Ah, I think this was left in during testing. I don't think it should be a problem to process the messages first. Let me try to remove it.